cfme/configure/access_control.py

```diff
@@ -237,25 +238,42 @@ def remove_tag(self, tag, value):
         flash.assert_success_message('Tag edits were successfully saved')
 
     @property
+    def exists(self):
+        try:
+            sel.force_navigate("cfg_accesscontrol_user_ed", context={"user": self})
+        except CandidateNotFound:
+            return False
+        else:
```

Comment:
If you take it out, you save one line of code and make it a little bit more straight-forward :) Up to you.